src/core/transform.js

```diff
@@ -61,7 +128,16 @@ p5.prototype.pushMatrix = function() {
  * @example
  * <div>
  * <code>
- * // Example in the works.
+ * translate(50, 50);
+ * applyMatrix(1, 0, 0, 0, 0, 0);
+ * rect(0, 0, 20, 20);
+ * // The matrix here is degenerate, it is
```

Comment:
This is rather confusing, I don't know what degenerate here means nor why it is impossible to draw anything with a y-dimension and I doubt users with less experience with transformation matrix will understand this either. 

I think simply doing a `translate` -> `rect` -> `resetMatrix` -> `rect` can be enough to illustrate the functionality of `resetMatrix()`